### PR TITLE
Draft of section 2

### PR DESCRIPTION
**Pull request by emile-00**
WIP currently

### CODE REVIEW

**Review by st1page — COMMENTED** [review]
The three sections seem to actually be part of an entire offloading scenario. Can we describe an overall explanation it in the readme.md and draw some architectural diagrams? In addition, we can explain to users how it was used without RisingWave. For example, compare:
1. Executing aggregate queries directly within the user's TP database. In this case, valuable TP resources are consumed. You can refer to the section 'The Problem: Running Complex Queries on OLTP' in this article.
2. Offloading to an OLAP system, such as ClickHouse. We might temporarily not complete this part. Because explaining the advantages in this situation is quite complex. Simply put, because RisingWave utilizes the pre-defined computations provided by the user, completing the computation at the time of data writing rather than at the time of querying, this allows for the reuse of repeated computations between multiple queries.

Furthermore, I think we can add a section on sinking the materialized view results of RisingWave back into PG. Although we do not recommend this approach because it incurs additional overhead compared to querying RisingWave directly. I believe there are two main advantages to this.
1. Compared to FDW, storing the materialized view results as a table in PG makes the performance of querying this result table more predictable.
2. Sometimes, with the help of PG's ecosystem, it is possible to achieve some functionalities that RisingWave does not support. For example, PG is compatible with more upper layers (such as Hasura), or special indexes can be supported at the storage layer through PG's plugins."

File: 02-bring-analytics-closer-to-odb/001-create-mv-offload-analytics.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Additionally, RisingWave must be configured to read data from a PostgreSQL table. To set up the PostgreSQL table and RisingWave table, follow the instructions outlined in [Ingest data from PostgreSQL CDC into RisingWave](/00-get-started/02-ingest-pg-cdc.md).
+
+## Create and update a materialized view
+
+Create a materialized view based on the CDC table, `pg_users`, created in Ingest data from PostgreSQL CDC into RisingWave. The materialized view `over21` will filter for all users who are at least 21 years old.
+
+```sql
+CREATE MATERIALIZED VIEW atleast21 AS
+SELECT * FROM pg_users
+WHERE age >= 21;
```
[the remark this answers is not in the document]
Comment (emile-00) [reply 02-bring-analytics-closer-to-odb/001-create-mv-offload-analytics.md]: Updated the query and data to fit a grouped aggregate query. Let me know if this meets the criteria.

**Comment by emile-00** [comment]
> The three sections seem to actually be part of an entire offloading scenario. Can we describe an overall explanation it in the readme.md and draw some architectural diagrams? In addition, we can explain to users how it was used without RisingWave. For example, compare:
> 
> 1. Executing aggregate queries directly within the user's TP database. In this case, valuable TP resources are consumed. You can refer to the section 'The Problem: Running Complex Queries on OLTP' in this article https://materialize.com/blog/oltp-workloads/.
> 2. Offloading to an OLAP system, such as ClickHouse. We might temporarily not complete this part. Because explaining the advantages in this situation is quite complex. Simply put, because RisingWave utilizes the pre-defined computations provided by the user, completing the computation at the time of data writing rather than at the time of querying, this allows for the reuse of repeated computations between multiple queries.
> 
> Furthermore, I think we can add a section on sinking the materialized view results of RisingWave back into PG. Although we do not recommend this approach because it incurs additional overhead compared to querying RisingWave directly. I believe there are two main advantages to this.
> 
> 1. Compared to FDW, storing the materialized view results as a table in PG makes the performance of querying this result table more predictable.
> 2. Sometimes, with the help of PG's ecosystem, it is possible to achieve some functionalities that RisingWave does not support. For example, PG is compatible with more upper layers (such as Hasura), or special indexes can be supported at the storage layer through PG's plugins."

I've added an extended overview to this tutorial section. I don't know if an architectural diagram would be that helpful in this case since we're only using RisingWave and PostgreSQL. 

I have a section on sinking data to PostgreSQL in part 3 and expanded on the benefits. 

Thanks for the quick feedback!

**Review by st1page — COMMENTED** [review]
LGTM, good job

**Pull request merged**